examples/PhasorDynamics/Example1/example1.cpp

```diff
@@ -141,22 +142,37 @@ int main()
     if (err > error_V)
       error_V = err;
 
-    std::cout << "GridKit: t = " << data.ti
-              << ", |V| = " << std::sqrt(data.Vr * data.Vr + data.Vi * data.Vi)
-              << ", w = " << (1.0 + data.dw) << "\n";
-    std::cout << "Ref    : t = " << ref_sol[0]
-              << ", |V| = " << ref_sol[2]
-              << ", w = " << ref_sol[1]
-              << "\n";
-    std::cout << "Error in |V| = "
-              << err
-              << "\n";
-    std::cout << "\n";
+    err = std::abs(1.0 + data.dw - ref_sol[1]) / (1.0 + ref_sol[1]);
+    if (err > error_w)
+      error_w = err;
+
+    // // Optional output
```

Comment:
Consider adding a verbosity flag and a conditional to get to this code. Or remove completely.

Reply:
Yes, that is my thinking. For that we need to move CLI utilities from Re::Solve to GridKit.